### DISCUSSION TITLE
Allow sponges to also work for lava

### DISCUSSION THREAD

**Discussion by RhythmicSys** [PurpurMC/Purpur#552] [__Completed__] [answered]
Used to have a lava sponge skript, was really nice, but super spammy haha

it'd be really nice especially with unlimited lava sources, I may be less afraid to turn that on 😆

**Comment by 12emin34** [comment, marked as answer]
Implemented in https://github.com/pl3xgaming/Purpur/commit/d54393f7c81873b06e3fb3f4caef971f430d3826
Discussion can be archived
